lib/cubits/authentication/authentication_cubit.dart

```diff
@@ -41,6 +44,9 @@ class AuthenticationCubit extends Cubit<AuthenticationState> {
 
   Future<void> unauthenticated() async {
     await _storage.clearAuthenticatedUser();
+    //Reset all cubits registered in the service locator. Otherwise their state is persisted to other user logins
+    sl<TicketsCubit>().resetCubit();
+    sl<ReceiptCubit>().resetCubit();
```

Comment:
Is this really the best practice? See https://stackoverflow.com/questions/60124310/how-to-reset-to-the-initial-state-value-whenever-i-close-my-app-using-flutter-bl

Reply:
#247 addresses this

Reply:
https://github.com/AnalogIO/coffeecard_app/pull/247 merged, fixed